src/main/java/billgates/view/gui/ActionPanel.java

```diff
@@ -244,6 +244,7 @@ public void view(UserJoinViewModel viewModel) {
             this.signInButton.setEnabled(false);
             this.signOutButton.setEnabled(true);
             this.addEntryButton.setEnabled(true);
+            this.backButton.setEnabled(true);
```

Comment:
Do we want to enable the `backButton` when the user joined the application? 
Or do we want to enable the `backButton` after we go into a split bill? 

In my understanding, we should enable back button when we're in split bill, then re-disable after we get out of split bill.

Reply:
Currently, we don't have a way for GUI to know the current bill.

We may improve this in the future. However, for now, we can simply enable it after the user login and disable it after logout.